src/status_im/ui2/screens/chat/composer/view.cljs

```diff
@@ -243,20 +241,27 @@
                                                             set-bg-opacity)]
               (quo.react/effect!
                #(do
```

Comment:
FYI: I know it's not the responsibility of this PR, but this usage of `#()` is a total anti-pattern in long, multi-line anonymous functions. In many occasions, it forces the reader to scan the entire code looking for a `%` sign that either doesn't exist or it's not supported by the anonymous function.